android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,3 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="app.edge.reactnative.currencyplugins">
```

Comment:
We can remove the `package=` prop once we add it to the build.grade.

Reply:
Hmm, this is not how it is configured in accountbased.